cookbooks/bcpc-hadoop/recipes/oozie_pack.rb

```diff
@@ -0,0 +1,98 @@
+#
+# Recipe to include hbase jar files into Oozie
+#
+%w{
+hbase
+}.each do |p|
+  package p do
+    action :install
+  end
+end
+
+file "/tmp/oozie.war" do
+  owner 'root'
+  group 'root'
+  mode 0744
+  content ::File.open("/usr/hdp/2.2.0.0-2041/oozie/oozie-server/webapps/oozie.war").read
+  action :create
+end
+
+directory '/tmp/oozie-hbase-prep' do
+  owner 'root'
+  group 'root'
+  mode '0755'
+  action :create
+end
+
+bash 'extract-oozie-war' do
+  user "root"
+  group "root"
+  code <<-EOH
+    cd /tmp/oozie-hbase-prep
+    jar -xf ../oozie.war
+  EOH
+  action :run
+end
+
+bash 'copy-hbase-jars' do
```

Comment:
Every time the machine restarts this will cause us to rebuild the Oozie WAR since this uses `/tmp`. Could you use a less raceable directory choice? (E.g. [Dir.mktmpdir](http://www.rubydoc.info/stdlib/tmpdir/Dir) or `Chef::Config[:file_cache_path]`.)